server/migration/main.go

```diff
@@ -0,0 +1,83 @@
+package migration
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/standup-raven/standup-raven/server/config"
+	"github.com/standup-raven/standup-raven/server/logger"
+	"github.com/standup-raven/standup-raven/server/standup"
+	"github.com/standup-raven/standup-raven/server/util"
+)
+
+var(
+	databaseSchemaVersion = "database_schema_version"
+	VERSION_1_4_0 = "1.4.0"
+	VERSION_1_5_0 =	"1.5.0"
+) 
+
+func DatabaseMigration() error {
+	key,err := config.Mattermost.KVGet(util.GetKeyHash(databaseSchemaVersion))
+	if err!=nil {
+		logger.Error("Couldn't fetch database schema version from KV store", err, nil)
+		return err
+	}
+	if key == nil {
+		version, err := json.Marshal(VERSION_1_4_0)
+		if err != nil {
+			logger.Error("Couldn't marshal database schema version", err, nil)
+			return err
+		}
+		if appErr := config.Mattermost.KVSet(util.GetKeyHash(databaseSchemaVersion), version); appErr != nil {
+			logger.Error("Couldn't update database version into KV store", appErr, nil)
+			return errors.New(appErr.Error())
+		}
+	}
+	upgradeDatabaseToVersion15()
```

Comment:
Instead of adding all guture upgrade calls over here, how about moving it to an `upgrade()` function?